modules/free/arrow-free-data/src/test/kotlin/arrow/free/FreeTest.kt

```diff
@@ -60,6 +62,7 @@ class FreeTest : UnitSpec() {
       EqLaws.laws(EQ) { Ops.value(it) },
       MonadLaws.laws(Ops, EQ),
       MonadLaws.laws(Free.monad(), EQ)
+       //FoldableLaws.laws(Free.foldable(), { Free.just<S,Int>(it) }, Eq.any())
```

Comment:
Should this be testing the TraverseLaws?

Reply:
Yes, once I get the code running